doc/source/point-in-time-recovery.rst

```diff
@@ -0,0 +1,85 @@
+.. _pitr:
+
+Point-in-Time Recovery
+********************************************************************************
+
+|PITR| is restoring a database up to a specific moment. |PITR| includes restoring the data from a backup snapshot and replaying all events that occurred to this data up to a specified moment from :term:`oplog slices <Oplog slice>`. |PITR| helps you prevent data loss during a disaster such as crashed database, accidental data deletion or drop of tables, unwanted update of multiple fields instead of a single one.
+
+|PITR| is available in |PBM| starting from v1.3.0. |PITR| is enabled via the ``pitr.enabled`` config option.
+
+.. code-block:: bash
+
+   $ pbm config --set pitr.enabled=true
+
+Incremental backups
+================================================================================
+
+When |PITR| is enabled, |pbm-agent| periodically saves consecutive slices of the :term:`oplog <Oplog>`. A method similar to the way replica set nodes elect a new primary is used to select the |pbm-agent| that saves the oplog slices. (Find more information in :ref:`pbm.architecture.agent`.)
+
+A slice covers a 10 minute span of oplog events. It can be shorter if |PITR| is disabled or interrupted by the start of a backup snapshot operation.
+
+The oplog slices are stored in the :file:`pbmPitr` subdirectory in the :ref:`remote storage defined in the config <pbm.config.initialize>`. A slice name reflects the start and end time this slice covers. 
+
+The |pbm-list| output includes both backup snapshots and valid time ranges for recovery. It also shows the |PITR| status.
+
+.. code-block:: bash
+
+   $ pbm list
+
+   Backup snapshots:
+  	2020-07-10T12:19:10Z
+  	2020-07-14T10:44:44Z
+  	2020-07-14T14:26:20Z
+  	2020-07-17T16:46:59Z
+   PITR <on>:
+  	2020-07-14T14:26:40 - 2020-07-16T17:27:26
+  	2020-07-17T16:47:20 - 2020-07-17T16:57:55
+
+.. note::
+
+   If you just enabled |PITR|, the time range list in the ``pbm list`` output is empty. It requires 10 minutes for the first chunk to appear in the list.  
+
+Restore to the point in time
+============================
+
+A restore and oplog updates are incompatible operations and cannot be run simultaneously. You must disable |PITR| before restoring a database:
```

Comment:
```suggestion
A restore and |PITR| incremental backups are incompatible operations and cannot be run simultaneously. You must disable |PITR| before restoring a database:
```

Reply:
Along with the edit suggestion above I realized something. The "Restoring a backup" section in running.rst should also advise that users should disable PITR first before running pbm restore.